### ISSUE TITLE
Queries column in Search Volumes

### ISSUE THREAD

**Issue by joshtorres** [tipoff/seo#96]
@drewroberts what is the queries column in the search_volumes table supposed to be for? It's an INT that isn't nullable. Should impressions go in here?
@phuclh

**Comment by drewroberts** [comment]
Yes, that is the impressions or search volume amount. I'm fine with changing the name if there's a better one you want to use.
